orttraining/orttraining/python/training/ortmodule/_custom_autograd_function_runner.py

```diff
@@ -93,6 +95,19 @@ def register_context(result):
             if training_mode_flag:
                 # Must extract one valid context from result tensors.
                 assert ctx is not None
+
+                #         FORWARD                                                    BACKWARD FUNCTION CONNECTIONS
+                # input_1 (leaf, constructed by from_dlpack)   <----reference----  AccumulateGrad gradient function
```

Comment:
Maybe also add some links to Pytorch code on github for the referred lines.

Reply:
yes will add. :)